examples/async-sqlite/README.md

```diff
@@ -49,7 +49,7 @@ A [`Root`][root] is a special handle to a JavaScript value that prevents the val
 
 [thread]: https://doc.rust-lang.org/std/thread/
 [mpsc]: https://doc.rust-lang.org/std/sync/mpsc/index.html
-[jsbox]: https://docs.rs/neon/0.8.1-napi/neon/types/struct.JsBox.html
-[eventqueue]: https://docs.rs/neon/0.8.1-napi/neon/event/struct.EventQueue.html
-[handle]: https://docs.rs/neon/0.8.1-napi/neon/handle/struct.Handle.html
-[root]: https://docs.rs/neon/0.8.1-napi/neon/handle/struct.Root.html
+[jsbox]: https://docs.rs/neon/0.9.1/neon/types/struct.JsBox.html
```

Comment:
Since the Node-API backend is now the default for docs, these can use `/neon/latest` instead of a specific version.

```suggestion
[jsbox]: https://docs.rs/neon/latest/neon/types/struct.JsBox.html
```